### ISSUE TITLE
version is hardocoded

### ISSUE THREAD

**Issue by edublancas** [ploomber/core#10]
this should be package, specific:

https://github.com/ploomber/core/blob/47e2db45740484ad862dc42c496f7c48f2c9f2f7/src/ploomber_core/telemetry/telemetry.py#L48

**Comment by idomic** [comment]
@edublancas this is so we know the telemetry version running with the product right?
like if it'll say ploomber 0.20 how would you know which telemetry is it?

**Comment by edublancas** [comment]
yeah exactly. I think the purpose of posthog's telemetry version is to version schema changes. right now, all packages will say `0.3`, but they should have independent versions as the telemetry schema for each will be different, essentially we want the Telemetry class to take this as an argument

to simplify things, we should make it an optional argument so that when we release this new version; it doesn't break existing packages. then open a PR in ploomber and set the value to `0.3`- however, that ploomber PR is currently blocked by https://github.com/ploomber/ploomber/pull/960 - we need to merge that first
